### PR TITLE
Logic: Add sort command

### PR DESCRIPTION
**Pull request by karrui**
Recreating this PR due to commits not updating due to changes to the repo settings

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.commands;
+
+/**
+ * Sorts all persons in the address book by alphabetical order of contact's name and lists to the user.
+ * The order can be either ascending or descending.
+ */
+public class SortCommand extends UndoableCommand {
```
Comment (yewshengkai) [inline src/main/java/seedu/address/logic/commands/SortCommand.java]: You have undo-able included for Sort as well? Nice.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -108,4 +109,21 @@ public static Index parseIndex(String oneBasedIndex) throws IllegalValueExceptio
...
     }
+
+    /**
+     * Parses {@code sortOrder} into a {@code boolean} and returns it.
+     * @throws IllegalValueException if the specified parameter is invalid (not "a", "d", or "").
+     */
+    public static boolean parseSort(String sortOrder) throws IllegalValueException {
+        String trimmedSortParameter = sortOrder.trim();
+        switch (trimmedSortParameter) {
+        case "":
```
Comment (yewshengkai) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java]: I assume the case "" here is equals to ascending order by default right?

Comment (karrui) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java]: Yup

Comment (yewshengkai) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java]: oki~

**Pull request merged**